hr_holidays_validate_days/models/hr_leave.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, api, _
+from odoo.exceptions import UserError
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class HrLeave(models.Model):
+    _inherit = 'hr.leave'
+
+    @api.multi
+    def action_validate_number_of_days(self):
+        def validate_number_of_days(leave):
```

Comment:
Can you put this method or the domain into class level so it can be inherited.

Reply:
Sure, done